uint32 x
uint32 y
---
bool success
float32 value
